vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml

```diff
@@ -496,40 +471,28 @@
     failed_when: upgraded_cp_nodes_count.stdout|int != 0
 
   - name: Get control plane machines
-    shell: | 
-            kubectl get machines -n "{{ NAMESPACE }}" -l  cluster.x-k8s.io/control-plane -o json |
-            jq -r '[ .items[] | select(.spec.version == "{{ UPGRADED_K8S_VERSION }}") | .status.nodeRef.name ]'
```

Comment:
you need the jq line ?It  was making sure you fetch upgraded controlplane name?

Reply:
yes, we need that for sure. Will revert this